static/js/tools/map/fetcher/default_stat.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch the default (best available) stat data
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import { Dispatch } from "react";
+
+import {
+  EntityObservationWrapper,
+  PointApiResponse,
+} from "../../../shared/stat_types";
+import { stringifyFn } from "../../../utils/axios";
+import { ChartDataType, ChartStoreAction } from "../chart_store";
+import { getDate } from "../util";
+
+export function fetchDefaultStat(
```

Comment:
fetchMapPointStat is kinda of different now, as I missed the map point sv and map point place type.

In general, I feel it's okay for repetition here since the type of data is limited and it allows update independently.